src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -443,3 +445,41 @@ def generate_maps(
     #   }
 
     return maps
+
+
+def subset_results_by_state(processed_results_dir: str, state: str) -> None:
+    # Loads final results and subsets those files to a provide state excluding the Social Security Observer
+
+    from vivarium_census_prl_synth_pop.constants.metadata import US_STATE_ABBRV_MAP
+
+    abbrev_name_dict = {v: k for k, v in US_STATE_ABBRV_MAP.items()}
+    state_name = sanitize_location(abbrev_name_dict[state.upper()])
+    processed_results_dir = Path(processed_results_dir)
+    all_results = processed_results_dir / "usa"
+    state_dir = processed_results_dir / "states" / state_name
+    state_dir.mkdir(exist_ok=True, parents=True)
+    # copy files from final results to state directory for further processing
+    shutil.copytree(all_results, state_dir, dirs_exist_ok=True)
+
+    for observer in FINAL_OBSERVERS:
+        if observer == "social_security_observer":
+            shutil.rmtree(state_dir / observer)
+        logger.info(f"Processing data for {observer}...")
```

Comment:
Nit: move this logging message above the if block for SS observer. 

Also add a log message that says we are not copying the SS observer data to the state results (I know we're technically copying it and then deleting it, but that's an implementation detail).